### PR TITLE
add office-file-size-limit config

### PR DESCRIPTION
**Pull request by zMingGit**

### CODE REVIEW

File: config/seafevents-conf.md [hunk 1]
```diff
@@ -22,6 +22,10 @@ interval=10m
 ## Refer to file search manual for details.
 index_office_pdf=false
 
+## The default size limit for doc, docx, ppt, pptx, xls, xlsx and pdf files, if smaller than the actual file, 
+## wiil result in a highlighted match text can't be displayed.This configuration takes effect in version 6.2.
```
Comment (shoeper) [inline config/seafevents-conf.md]: wiil -> it will

Space missing before This

File: config/seafevents-conf.md [hunk 2]
```diff
@@ -22,6 +22,10 @@ interval=10m
...
+office_file_size_limit: 29000000
```
Comment (shoeper) [inline config/seafevents-conf.md]: is it in bytes?

File: config/seafevents-conf.md [hunk 3]
```diff
@@ -23,8 +23,9 @@ interval=10m
 index_office_pdf=false
 
 ## The default size limit for doc, docx, ppt, pptx, xls, xlsx and pdf files, if smaller than the actual file, 
-## wiil result in a highlighted match text can't be displayed.This configuration takes effect in version 6.2.
-office_file_size_limit: 29000000
+## it wiil result in a highlighted match text can't be displayed. This configuration takes effect in version 6.2.
```
Comment (shoeper) [inline config/seafevents-conf.md]: wiil -> will

Comment (freeplant) [reply config/seafevents-conf.md]: You review on an outdated version.

**Pull request merged**